addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxControlTimedSwitch.java

```diff
@@ -87,7 +87,7 @@ public static boolean accepts(String type) {
      *             when something went wrong with communication
      */
     public void on() throws IOException {
-        socketClient.sendAction(uuid, CMD_ON);
+        socketClient.sendAction(uuid, CMD_PULSE);
```

Comment:
This is confusing, it should be pulse() not on(), because there is still valid ON command that just sets the switch to on permanently. I would add on() and off() commands to this file, even if they are not used by the handler, because this is how it is done in all other controls, for API completeness in .core package.